powerstore/volume_helper.go

```diff
@@ -287,9 +299,11 @@ func modifyVolume(planVol models.Volume, valInBytes int64, volID string, client
 	vgModify := &pstore.VolumeModify{
 		Name:                planVol.Name.ValueString(),
 		Size:                valInBytes,
-		ProtectionPolicyID:  &protectionPolicy,
+		ProtectionPolicyID:  protectionPolicy,
```

Comment:
@AnikaAgiwal2711 this is the change where "&protectionPolicy" is changed to "protectionPolicy" as it updated from a pointer to normal variable on gopowerstore side through https://github.com/dell/gopowerstore/pull/99/commits/1c5d1d09f5b464b78d13ed0d6811a02f8c4e4504

Reply:
Thanks